organization/components/roles/RoleTable.tsx

```
@@ -0,0 +1,308 @@
+                          )}
+                        </td>
+                      )}
+                    </tr>
+                    {expandedRow === role._id && (
+                      <tr className="bg-blue-50">
+                        <td colSpan={canDelete ? 5 : 4} className="px-6 py-4">
+                          <div className="mb-3">
+                            <h4 className="text-sm font-medium text-gray-900 mb-2">
+                              Full Role Details
+                            </h4>
+                            <div className="text-sm text-gray-600 mb-2"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)